src/store/auths/authsActions.ts

```diff
@@ -81,7 +83,9 @@ export const authAdd = (nickname: string, privateKey: string, passphrase: string
         throw new Error('Invalid Passphrase')
       }
 
-      if (!isValidPrivate(privateKey)) {
+      try {
+        PrivateKey.fromString(privateKey).toElliptic()
```

Comment:
it'll fail on the first part; you can remove .toElliptic()

Reply:
https://github.com/EOSIO/eosio-reference-chrome-extension-authenticator-app/pull/37/commits/d8455f0e00255a5be1781e1643d298daf1572df3#diff-b6bf3919e763d246ff62d57e2bcbc07b